src/lookAheadSuggestion.ts

```diff
@@ -39,19 +39,23 @@ window.onDidChangeTextEditorSelection(clearCurrentLookAheadSuggestion);
 export async function getLookAheadSuggestion(
   document: TextDocument,
   { range, text }: SelectedCompletionInfo,
-  position: Position
+  position: Position,
+  cancellationToken: CancellationToken
 ): Promise<InlineCompletionList<TabnineInlineCompletionItem>> {
-  const isContainsCompletionInfo = text.startsWith(document.getText(range));
+  const textAtRange = document.getText(range);
+  const isContainsCompletionInfo = text.startsWith(textAtRange);
 
   if (!isContainsCompletionInfo) {
     return new InlineCompletionList([]);
   }
-
-  const response = await retry(
-    () => runCompletion(document, range.end, undefined, text),
-    (res) => !!res?.results.length,
-    2
-  );
+  const response = await runCompletion({
+    document,
+    position: range.end,
+    currentSuggestionText: text.substring(textAtRange.length),
```

Comment:
this fixes another issue which is tested here:
https://github.com/codota/tabnine-vscode/pull/1092/files#diff-b2e71e49777c991cd78d2cb59dbe42c942ebac2d67af481197848db7f90540a1R355

Reply:
commit https://github.com/codota/tabnine-vscode/pull/1092/commits/cd3f5d84a6e93cc04599a7132dc00b258d97beaa